.github/actionlint-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+      {
+        "owner": "actionlint",
+        "pattern": [
+          {
+            "regexp": "^(?:\\x1b\\[\\d+m)?(.+?)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*:(?:\\x1b\\[\\d+m)*(\\d+)(?:\\x1b\\[\\d+m)*: (?:\\x1b\\[\\d+m)*(.+?)(?:\\x1b\\[\\d+m)* \\[(.+?)\\]$",
```

Comment:
That is quite the regular expression. No way to make it a bit clearer to read? I know that unfortunately we cannot add comments to it since it's a JSON file.

Reply:
I just copied what they said to include, it does seem excessive I agree!

Reply:
Well, if we never have to update ourselves it'll be fine...